tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -55,12 +55,13 @@ export class ScalarCardDataTable {
   getMinValueInRange(
     points: ScalarCardPoint[],
     startPointIndex: number,
-    endPointIndex: number
+    endPointIndex: number,
+    smoothed: boolean = false
```

Comment:
Why we want to make this an input? (Instead of just updating every point.value to point.y) Is the idea to reuse this method twice?